scripts/traverse.js

```diff
@@ -46,11 +46,11 @@ function traverseFeatures(obj, depth, identifier) {
   }
 }
 
-let features = [];
-let folders = Array.isArray(argv.folder) ? argv.folder : argv.folder.split(",");
-let values = Array.isArray(argv.value) ? argv.value : argv.value.toString().split(",");
+const features = [];
```

Comment:
`features` should be `let` as we're pushing to it.  (I probably should have put it towards the top of the file as well, but that's something for another time!)